src/dbnode/storage/mediator.go

```diff
@@ -147,7 +144,11 @@ func (m *mediator) Open() error {
 	go m.ongoingFileSystemProcesses()
 	go m.ongoingColdFlushProcesses()
 	go m.ongoingTick()
-	m.databaseRepairer.Start()
+
+	for _, process := range m.backgroundProcesses {
+		go process.Run()
```

Comment:
Is it correct that this doesn't pick up any newly registered background processes?

Reply:
Correct (as we don't have a use case for that). I will add this to the comment of `RegisterBackgroundProcess` just in case.